x/profiles/types/keys.go

```diff
@@ -133,3 +137,18 @@ func UserApplicationLinkKey(user, application, username string) []byte {
 func ApplicationLinkClientIDKey(clientID string) []byte {
 	return append(ApplicationLinkClientIDPrefix, []byte(clientID)...)
 }
+
+// ExpiringApplicationLinkPrefix returns the prefix used to store client id keys of the expiring application links
+// at the given block height
+func ExpiringApplicationLinkPrefix(blockHeight int64) []byte {
```

Comment:
I think this can be simply written like this:
```golang
buf := make([]byte, binary.MaxVarintLen64)
binary.BigEndian.PutUint64(buf, uint64(blockHeight))
return append(ApplicationLinkExpirationPrefix, buf...)
```
as stated [here](https://stackoverflow.com/a/35371760) it will not affect the sign of the number